cloud-functions/cf-dsm-file-scanner/checkstyle.xml

```diff
@@ -24,12 +24,6 @@
         <property name="lineSeparator" value="lf"/>
     </module>
 
-    <module name="LineLength">
-        <property name="fileExtensions" value="java"/>
-        <property name="max" value="140"/>
-        <property name="ignorePattern" value="^package.*|^import.*|a href|href|http://|https://|ftp://"/>
-    </module>
```

Comment:
FWIW, every place I've worked had a line length restriction, so not having one is non-standard, and will probably decrease readability for those accustomed to restrictions. 
Is your concern that having this as part of the check style leads to errors, or do you prefer the readability of long lines over multi-line continuations? Or something else?